src/ga4gh/core/_internal/models.py

```diff
@@ -11,9 +11,14 @@
     module name, e.g., `ga4gh.core.core_models.Gene`
 """
 from typing import Any, Dict, List, Literal, Optional, Union
+try:
+    from typing import Annotated  # Python 3.9 and later
+except ImportError:
+    from typing_extensions import Annotated  # For Python 3.8
```

Comment:
@ga4gh/vrs-python-maintainers are we going to follow biocommons and only support 3 most recent versions of Python?

Reply:
I would vote for >=3.9

Python 3.8 official EOL is in a few months.